lib/zuora/rest.rb

```diff
@@ -3,6 +3,9 @@ module Zuora
   module Rest
     API_URL = 'https://api.zuora.com/rest/v1/'.freeze
     SANDBOX_URL = 'https://apisandbox-api.zuora.com/rest/v1/'.freeze
+    # Newest endpoints for REST API calls
+    BETA_API_URL = 'https://rest.zuora.com/v1/'.freeze
+    BETA_SANDBOX_URL = 'https://rest.apisandbox.zuora.com/v1/'.freeze
```

Comment:
1) These base endpoints no longer appear to be in beta as they are listed in the official [REST API doc](https://www.zuora.com/developer/api-reference/#section/Introduction/Endpoints).

2) In Zuora's latest API [release notes](https://community.zuora.com/t5/Developers/API-Changelog/gpm-p/18092), they have removed `v1` from the [base endpoints](https://www.zuora.com/developer/api-reference/#section/Introduction/Endpoints) and add the version to endpoint paths (from the doc => starting with this release `not all endpoints will contain  will contain “/v1”` - it would seem not all API endpoints will be versioned going forward).

Reply:
I think this PR is still safe, as it is backwards compatible but gives the needed functionality for now.